unskript-ctl/unskript_ctl_run.py

```diff
@@ -830,6 +856,13 @@ def get_first_cell_content(self):
         first_cell_content += '''w = Workflow(env, secret_store_cfg, None, global_vars=globals(), check_uuids=None)'''
         return first_cell_content
 
+    def get_timeout_decorator_function(self, execution_timeout):
+        with open(os.path.join(os.path.dirname(__file__), 'templates/timeout_handler.j2'), 'r') as f:
+            content_template = f.read()
+
+        template = Template(content_template)
+        return  template.render(execution_timeout=execution_timeout)
+    
```

Comment:
Is this really needed? Not sure what value its adding. We can just include the decorator definition in the jit file.